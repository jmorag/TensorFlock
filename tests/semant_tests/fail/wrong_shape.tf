main = tensor;

tensor : T<1,2>;
tensor = [1. 2. 3.];


tensor2 : T<4>;
tensor2 = [1. 2. 3.];
