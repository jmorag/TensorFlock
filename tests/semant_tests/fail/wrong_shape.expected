Fatal error: exception Failure("Declared type T<(1), (2)> but received type T<(3)>")
